src/components/Footer/Footer.module.scss

```diff
@@ -1,9 +1,104 @@
 .footer {
   position: relative;
-  flex-shrink: 0;
   padding: 1rem;
   background-color: $primary-color;
+  color: $secondary-color;
+  overflow: hidden;
+
+  @media (min-width: $screen-sm-min) {
+    padding: 4rem 2rem 2rem;
+  }
+}
+
+.mainContent {
+  display: grid;
+  grid-template-columns: 1fr;
+  max-width: 1200px;
```

Comment:
Whenever you have something global like this. Used 4 times, it might be a good time to abstract it. My rule of thumb is 3 uses.

![CleanShot 2025-01-21 at 18 20 04@2x](https://github.com/user-attachments/assets/306131ca-79a3-4a84-abe7-caf8dfa7d4d0)